pkg/proxy/ipvs/proxier.go

```diff
@@ -489,10 +489,8 @@ func NewDualStackProxier(
 	initOnly bool,
 ) (proxy.Provider, error) {
 
-	safeIpset := newSafeIpset(ipset)
```

Comment:
oh, so they share the same exec 
https://github.com/kubernetes/kubernetes/pull/122373/commits/b69510b0690073510fdb787f34687c4d0a40e358#r1441864562